cloudsync/event.py

```diff
@@ -25,10 +25,11 @@ class Event:
     oid: str                                    # fsobject id
     path: Optional[str]                         # path
     hash: Any                                   # fsobject hash     (better name: ohash)
-    exists: Optional[bool]
+    exists: Optional[bool]                      # True, False or None ... please do not assume None == False!
```

Comment:
probably a good idea to make this an enum to enforce this...

Reply:
yep

Reply:
backward breaking